.github/workflows/ci.yml

```diff
@@ -28,6 +28,14 @@ jobs:
     - name: Run All Validations
       if: ${{ matrix.node-version != '10.x' && matrix.node-version != '12.x' }}
       run: npm run ci
+    - name: Lint Dockerfile
+      if: ${{ matrix.os == 'ubuntu-latest' }}
+      run: |
+        sudo apt-get -qq update && sudo apt-get -qq -y --no-install-recommends install ca-certificates curl jq
+        LATEST_RELEASE=$(curl -L -s -H "Accept: application/json" "https://github.com/hadolint/hadolint/releases/latest" | jq -r '.tag_name')
+        curl -sL -o hadolint "https://github.com/hadolint/hadolint/releases/download/${LATEST_RELEASE}/hadolint-Linux-x86_64"
+        sudo chmod +x hadolint
+        ./hadolint docker/Dockerfile 
```

Comment:
depending on how strict you are with github workflow definition, there might be a simpler way to do this. If you agree with having a separate job to perform this step, we could use the available container image instead of downloading and installing `hadolint` (I'm less familiar with github workflow syntax...):

```yaml
jobs:
  lint-dockerfile:
    runs-on: ubuntu-latest
    container: hadolint/hadolint:latest-alpine
    steps:
    - uses: actions/checkout@v2
    - name: hadolint
    - run: hadolint docker/Dockerfile

```

Reply:
I like the dedicated job idea. Looks much cleaner. :) I’ve used that in a couple of other places.